### DISCUSSION TITLE
Double press of window keys behaves differently

### DISCUSSION THREAD

**Discussion by Kirinko** [qtile/qtile#3287] [Q&A]
Hi, I noticed a change in the behavior of Qtile since last update. Earlier, I could switch back to window I had jumped from simply by pressing the key combination of current window (i.e. I was on window 2, then pressed M+3 and worked in window 3. If I wanted to move back to window 2, I could just pres M+3 again). This idea still works with mouse navigation (if I press on icon of current window I'm moved to previous window). 
Is there some new configuration to enable this feature for key navigation?

**Comment by m-col** [comment]
Sounds like you are using `lazy.group.toscreen()`? If so, pass `toggle=True` to it and it will use the old behaviour.
